src/groupeddatatable/grouping.jl

```diff
@@ -201,7 +201,11 @@ function combine(ga::GroupApplied)
         idx[j + (1:n)] = gd.idx[start]
         j += n
     end
-    hcat!(gd.parent[idx, gd.cols], valscat)
+    if isa(valscat, DataTable)
```

Comment:
When I broke this during my changes it definitely helped understand how `combine` and my changes to `aggregate` in https://github.com/JuliaData/DataTables.jl/pull/65 differ. This `hcat!` call was handling both DataTables and Vectors